t/t2204-job-info.t

```diff
@@ -47,13 +56,9 @@ test_expect_success 'flux job eventlog works on multiple entries (active)' '
 	grep -q foo eventlog_b.out
 '
 
-# we cheat and manually move active to inactive in these tests
-
 test_expect_success 'flux job eventlog works (inactive)' '
         jobid=$(flux job submit test.json) &&
-        activekvsdir=$(flux job id --to=kvs-active $jobid) &&
-        inactivekvsdir=$(echo $activekvsdir | sed 's/active/inactive/') &&
-        flux kvs move ${activekvsdir}.eventlog ${inactivekvsdir}.eventlog &&
+        move_inactive $jobid &&
```

Comment:
should there be a `flux job wait-event <id> depend` (or whatever event) here to ensure the submitted job has reached its final state before moving?   Otherwise the eventlog may appear in both directories.

Reply:
i don't load sched-simple or jog-manager in these tests, so I think this is fine.  But just to be on the safe side for the future we could add a wait-event.  But what should we wait for to be safe?

Reply:
Right now there are two events: `submit` and `depend` but that behavior could change.  In particular once we have dependencies, the `depend` event won't happen until after the dependency manager responds, and if you don't load it here, it won't.

To avoid being fragile with respect to ongoing development in the job system, it might be best to cancel the job and wait for `finish`.  At that point you know there will be no more updates to the job.

(This pattern is repeated in other subtests and in the job-info-security test script)

Reply:
ahh, I now see that the `job-manager` module is now loaded in `t/rc/rc1-job`, which now makes this possible.

Reply:
I think it has been that way for some time.  In fact if you submit jobs while the job manager is not loaded, you will get an error on job submission.  (A misleading one it turns out - I will open an issue)

Reply:
ok, I think I'm just getting confused, b/c of some tests load the job modules (e.g. `t2202`).

Reply:
the `finish` event requires `sched-simple` to be loaded as well.  Perhaps we should wait for `clean`?

Reply:
Right! Sorry, that's what I meant.